diff --git a/proteus/mprans/ArgumentsDict.h b/proteus/mprans/ArgumentsDict.h
index 1111111..2222222 100644
--- a/proteus/mprans/ArgumentsDict.h
+++ b/proteus/mprans/ArgumentsDict.h
@@ -5,7 +5,7 @@
 #include <stdexcept>
 #include <string>
 #include "xtensor-python/pyarray.hpp"
-#include "xtensor/xio.hpp"
+#include "xtensor/io/xio.hpp"
 
 namespace proteus
 {
diff --git a/proteus/mprans/RANS2P.h b/proteus/mprans/RANS2P.h
index 1111111..2222222 100644
--- a/proteus/mprans/RANS2P.h
+++ b/proteus/mprans/RANS2P.h
@@ -10,9 +10,9 @@
 #include "PyEmbeddedFunctions.h"
 #include "equivalent_polynomials.h"
 #include "ArgumentsDict.h"
-#include "xtensor/xarray.hpp"
-#include "xtensor/xview.hpp"
-#include "xtensor/xfixed.hpp"
+#include "xtensor/containers/xarray.hpp"
+#include "xtensor/views/xview.hpp"
+#include "xtensor/containers/xfixed.hpp"
 #include "xtensor-python/pyarray.hpp"
 #include "mpi.h"
 #include "proteus_lapack.h"
diff --git a/proteus/mprans/RANS2P2D.h b/proteus/mprans/RANS2P2D.h
index 1111111..2222222 100644
--- a/proteus/mprans/RANS2P2D.h
+++ b/proteus/mprans/RANS2P2D.h
@@ -10,9 +10,9 @@
 #include "PyEmbeddedFunctions.h"
 #include "equivalent_polynomials.h"
 #include "ArgumentsDict.h"
-#include "xtensor/xarray.hpp"
-#include "xtensor/xview.hpp"
-#include "xtensor/xfixed.hpp"
+#include "xtensor/containers/xarray.hpp"
+#include "xtensor/views/xview.hpp"
+#include "xtensor/containers/xfixed.hpp"
 #include "xtensor-python/pyarray.hpp"
 #include "mpi.h"
 #include "proteus_lapack.h"
